ee/identitymanager/identity_managers/keycloak/keycloak_authverifier.py

```diff
@@ -55,6 +55,8 @@ def _verify_bearer_token(
         email = payload.get("preferred_username")
         org_id = payload.get("active_organization", {}).get("id")
         org_realm = payload.get("active_organization", {}).get("name")
+        if org_realm is None:
+            org_realm = self.keycloak_client.realm_name
```

Comment:
it shouldn't happen :(

Reply:
it basically means the org mapper doesn't work

Reply:
Removed from this PR.